src/GradeBook.tsx

```diff
@@ -5,23 +5,27 @@ import { Auth } from 'aws-amplify';
 import HeaderComponent from './components/Header';
 import ProfileDropdown from 'molecules/ProfileDropdown';
 import { getCurrentSemester, Semester } from 'api';
+import GradeBookContentContainer from './components/GradeBookContentContainer';
+import GradeBookHeader from 'components/GradeBookHeader';
+// todo change mock data
+import { Fall2020 } from 'api/mock';
 
 const GradeBookContainer = styled.div`
-	height: 100vh;
+	height: 100%;
```

Comment:
Ah, I suspect it is because it would not allow for scrolling... What we probably (maybe) want is a `min-height: 100vh` and `height: 100%` so that it will at least be as big as the page, but allows for scrolling. That being said, if it scrolls we want to make sure the rest of the page looks as expected (still see top bar and sidebar covers whole side). @cindyunrau can you comment

Reply:
yes thats right, it wouldnt scroll with a static height, the min height is a good idea though, i will implement that